### PR TITLE
SWATCH-2018: Report filtered instance data to Export Service in JSON

### PR DESCRIPTION
**Pull request by Sgitario**
Jira issue: [SWATCH-2018](https://issues.redhat.com/browse/SWATCH-2018)

## Description
- configured the export listener for the tally service (worker profile)
- report the instances data as JSON
- generated instances export POJOs from JSON schemas

Note that these changes include a refactor to reuse the existing exporter listener logic for both the subscription sync and the tally services. You can see the refactor and the actual feature into the different commits to easier the code review.

## Testing
1.- start kafka and postgresql: `podman-compose up -d`
2.- start export-service: `podman-compose -f config/export-service/docker-compose.yml up -d`
3.- start the tally service: `RHSM_RBAC_USE_STUB=true DEV_MODE=true SPRING_PROFILES_ACTIVE="worker,api,kafka-queue" ./gradlew :bootRun`
4.- load some data:

```
insert into account_services(org_id, service_type) values ('123', 'test');

insert into hosts(id, org_id, num_of_guests, instance_id, display_name, billing_provider, billing_account_id, instance_type) 
values ('b870dc34-53a3-4634-bfd1-56c227bc36ac','123', 3, '456', 'my host', 'AWS', '789', 'test');

insert into host_tally_buckets (host_id, product_id, "usage", sla, as_hypervisor, billing_provider, billing_account_id, measurement_type, cores, sockets) 
values ('b870dc34-53a3-4634-bfd1-56c227bc36ac', 'RHEL for x86', 'PRODUCTION', 'PREMIUM', true, 'AWS', '789', 'PHYSICAL', 3, 6);

insert into instance_measurements (host_id, metric_id, value)
values ('b870dc34-53a3-4634-bfd1-56c227bc36ac', 'SOCKETS', 4.8);
```

5.- create a new export request from Export Service:

get the identifier first:
```
echo -n '{"identity":{"account_number":"10001","org_id":"123","internal":{"org_id":"13259775"},"type":"User","user":{"username":"user_dev"}}}' | base64 -w 0
> eyJpZGVudGl0eSI6eyJhY2NvdW50X251bWJlciI6IjEwMDAxIiwib3JnX2lkIjoiMTIzIiwiaW50ZXJuYWwiOnsib3JnX2lkIjoiMTMyNTk3NzUifSwidHlwZSI6IlVzZXIiLCJ1c2VyIjp7InVzZXJuYW1lIjoidXNlcl9kZXYifX19
```

call the export service:
```
curl -sS -X POST http://localhost:8002/api/export/v1/exports -H "x-rh-identity: eyJpZGVudGl0eSI6eyJhY2NvdW50X251bWJlciI6IjEwMDAxIiwib3JnX2lkIjoiMTIzIiwiaW50ZXJuYWwiOnsib3JnX2lkIjoiMTMyNTk3NzUifSwidHlwZSI6IlVzZXIiLCJ1c2VyIjp7InVzZXJuYW1lIjoidXNlcl9kZXYifX19" -H "Content-Type: application/json" -d '{
    "name": "Test Export Request",
    "format": "json",
    "expires_at": "2025-01-01T00:00:00Z",
    "sources": [
        {
            "application": "subscriptions",
            "resource": "instances",
            "filters": {
                "product_id": "RHEL for x86"
            }
        }
    ]
}'
```

Expected output is:
```
{"id":"78244e87-2abe-49af-8821-67b14128c5c3","created_at":"2024-04-05T06:30:01.034156787Z","expires_at":"2025-01-01T00:00:00Z","name":"Test Export Request","format":"json","status":"pending","sources":[{"id":"2008873d-916d-4240-8cd8-55ca00141c55","application":"subscriptions","status":"pending","resource":"instances","filters":{"product_id":"RHEL for x86"}}]}
```

6.- verification:

check the tally logs where you should see the new event coming from the Export service topic:
```
2024-04-05 07:09:15,463 [thread=swatch-subscription-export-0-C-1] [INFO ] [org.candlepin.subscriptions.export.ExportSubscriptionListener] - Processing event: 'd3451fe1-3951-4914-b679-28f7594033e7' from application: 'urn:redhat:source:console:app:export-service'
2024-04-05 07:09:15,652 [thread=swatch-subscription-export-0-C-1] [INFO ] [org.candlepin.subscriptions.export.ExportSubscriptionListener] - Event processed: 'd3451fe1-3951-4914-b679-28f7594033e7' from application: 'urn:redhat:source:console:app:export-service'
```

The export-service should have uploaded the generated report by the subscription sync service into S3 (minio instance). 
You can verify that this report has been uploaded by checking the folder 'config/export-service/tmp/minio/exports-bucket' where should be a file at `<BUCKET ID>.json/xl.meta`. The `xl.meta` file is binary, but if you open the txt plain editor, you should see:

```
{"data":[{"id":"b870dc34-53a3-4634-bfd1-56c227bc36ac","instance_id":"456","display_name":"my host","billing_provider":"aws","billing_account_id":"789","measurements":[{"metric_id":"Sockets","value":6.0},{"metric_id":"SOCKETS","value":4.8}],"number_of_guests":3,"category":"physical"}]}
```

7.- verify wrong filters:

let's try to use a wrong value for product ID:

```
curl -sS -X POST http://localhost:8002/api/export/v1/exports -H "x-rh-identity: eyJpZGVudGl0eSI6eyJhY2NvdW50X251bWJlciI6IjEwMDAxIiwib3JnX2lkIjoiMTMyNTk3NzUiLCJpbnRlcm5hbCI6eyJvcmdfaWQiOiIxMzI1OTc3NSJ9LCJ0eXBlIjoiVXNlciIsInVzZXIiOnsidXNlcm5hbWUiOiJ1c2VyX2RldiJ9fX0=" -H "Content-Type: application/json" -d '{
    "name": "Test Export Request",
    "format": "json",
    "expires_at": "2025-01-01T00:00:00Z",
    "sources": [
        {
            "application": "subscriptions",
            "resource": "instances",
            "filters": {
                "product_id": "Wrong!"
            }
        }
    ]
}'
```

Then, you should see the following traces in the the subscription sync logs:

```
2024-04-05 07:11:33,721 [thread=swatch-subscription-export-0-C-1] [ERROR] [org.candlepin.subscriptions.export.ExportSubscriptionListener] - Error thrown for event: 'b2af53cc-d8d4-462f-a12e-91b70a65bcfa' sending ErrorRequest: 'Wrong filter in export request: ProductId: Wrong! not found in configuration'
```

And in the export-service database (by default, it's: "localhost:5342", user: postgres, password: postgres, database name: postgres), you can see a new record in the table "sources" with:

```
345a3009-682e-4281-b0ce-becc2c6e47a0	675a00d0-98b4-4d1f-8f10-6632ded1e214	subscriptions	failed	instances	{"product_id": "Wrong!"}	400	Wrong filter in export request: ProductId: Wrong! not found in configuration
```

### CODE REVIEW

**Comment by san7ket** [comment]
For a valid product_id, 
```

In [32]: product_id="OpenShift-dedicated-metrics"

In [33]: cluster=app.rhsm_subscriptions.create_mock_payg_cluster(product_id=product_id)
2024-04-11 14:32:13.490 [    INFO] [root] Adding payg events of org 17790802 for clowder_smoke env ...
2024-04-11 14:32:13.514 [    INFO] [root] POST request response url: http://swatch-tally-service:8000/api/rhsm-subscriptions/v1/internal/rpc/tally/events, status_code: 200
2024-04-11 14:32:13.515 [    INFO] [root] Response: {
    "detail": "Events saved"
}
2024-04-11 14:32:13.515 [    INFO] [root] OpenShift-dedicated-metrics PAYG Cluster 9afb8efe-be7a-4489-a04e-6b3a3574d2bf created successfully!!

In [34]: application.rhsm_subscriptions.sync_tally_hourly(
    ...:         hours=3, wait=True, product_id=product_id, perform_metering=False
    ...:     )
2024-04-11 14:32:24.574 [    INFO] [iqe.base.rest_client] REST: METHOD=GET, request_id=None, params=[('granularity', 'Daily'), ('beginning', '2024-04-10T14:07:17.830047+00:00'), ('ending', '2024-04-11T14:32:24.552498+00:00')]
2024-04-11 14:32:24.575 [    INFO] [root] {'date': datetime.datetime(2024, 4, 11, 0, 0, tzinfo=tzlocal()),
 'has_data': False,
 'value': 0}
2024-04-11 14:32:24.587 [    INFO] [root] POST request response url: http://swatch-tally-service:8000/api/rhsm-subscriptions/v1/internal/tally/hourly?org=17790802&start=2024-04-11T12%3A00%3A00Z&end=2024-04-11T15%3A00%3A00Z, status_code: 204
2024-04-11 14:32:24.587 [    INFO] [root] Not valid json output. Returning plain text message.
2024-04-11 14:32:24.587 [    INFO] [root] from 2024-04-11T12:00:00Z to 2024-04-11T15:00:00Z.
2024-04-11 14:32:24.600 [    INFO] [root] Waiting for hourly tally sync completion...
2024-04-11 14:32:39.634 [    INFO] [iqe.base.rest_client] REST: METHOD=GET, request_id=None, params=[('granularity', 'Daily'), ('beginning', '2024-04-10T14:07:17.830047+00:00'), ('ending', '2024-04-11T14:32:39.615087+00:00')]
2024-04-11 14:32:39.635 [    INFO] [root] {'date': datetime.datetime(2024, 4, 11, 0, 0, tzinfo=tzlocal()),
 'has_data': True,
 'value': 1}

In [35]: new_systable = application.rhsm_subscriptions.search_system_table(
    ...:         product_id=product_id, display_name_contains=cluster
    ...:     )
2024-04-11 14:32:46.595 [    INFO] [iqe.base.rest_client] REST: METHOD=GET, request_id=None, params=[('limit', 100), ('offset', 0), ('display_name_contains', '9afb8efe-be7a-4489-a04e-6b3a3574d2bf')]
2024-04-11 14:32:46.596 [    INFO] [root] [{'id': '36f4f918-78b8-43b7-b6ef-67a0a5f76f37', 'instance_id': '9afb8efe-be7a-4489-a04e-6b3a3574d2bf', 'display_name': 'automation_osd_cluster_9afb8efe-be7a-4489-a04e-6b3a3574d2bf', 'billing_provider': 'red hat', 'measurements': [1.0, 1.0], 'last_seen': datetime.datetime(2024, 4, 11, 13, 0, tzinfo=tzlocal()), 'last_applied_event_record_date': datetime.datetime(2024, 4, 11, 14, 32, 13, 507010, tzinfo=tzlocal()), 'number_of_guests': 0, '_labeled_measurements': {'Cores': 1.0, 'Instance-hours': 1.0}}]

In [36]: export = application.rhsm_subscriptions.create_swatch_export_request(
    ...:         filters={"product_id": f"{product_id}"}, resource="instances"
    ...:     )
2024-04-11 14:32:54.984 [    INFO] [iqe.base.rest_client] REST: METHOD=POST, request_id=None, params=[]
2024-04-11 14:32:54.985 [    INFO] [root] Waiting for export request with id e416fdd2-0274-42cd-bd2a-6aac3af28887 to process
2024-04-11 14:32:54.985 [    INFO] [root] {'completed_at': None,
 'created_at': datetime.datetime(2024, 4, 11, 14, 32, 54, 977937, tzinfo=tzlocal()),
 'expires_at': datetime.datetime(2024, 4, 18, 14, 32, 54, 977923, tzinfo=tzlocal()),
 'format': 'json',
 'id': 'e416fdd2-0274-42cd-bd2a-6aac3af28887',
 'name': 'IACHMOpwwG',
 'sources': [{'application': 'subscriptions',
              'filters': {'product_id': 'OpenShift-dedicated-metrics'},
              'id': 'cbe3bcdb-3cbe-4f31-be2a-f979e0a30f09',
              'resource': 'instances',
              'status': 'pending'}],
 'status': 'pending'}
```

The export request is stuck in pending state, I see log in export service , but I see no logs in tally-service pod, nor in subscription service pods 
Export pod logs:
```
{"level":"debug","@timestamp":"2024-04-11T14:32:54.9838Z","caller":"exports/request_resources.go:92","msg":"sending kafka message to the producer","request_id":"export-service-service-57496c48d9-bkw9t/eroqxBeKT2-000017","org_id":"17790802","export_id":"e416fdd2-0274-42cd-bd2a-6aac3af28887"}
{"level":"info","@timestamp":"2024-04-11T14:32:54.9839Z","caller":"exports/request_resources.go:94","msg":"sent kafka message to the producer: platform.export.requests[-1]@0","request_id":"export-service-service-57496c48d9-bkw9t/eroqxBeKT2-000017","org_id":"17790802","export_id":"e416fdd2-0274-42cd-bd2a-6aac3af28887"}
{"level":"info","@timestamp":"2024-04-11T14:33:15.0129Z","caller":"logger/logger.go:95","msg":"","protocol":"HTTP/1.1","request":"GET","path":"/api/export/v1/exports/e416fdd2-0274-42cd-bd2a-6aac3af28887/status","latency":0.00324921,"status":200,"size":372,"request_id":"export-service-service-57496c48d9-bkw9t/eroqxBeKT2-000018"}

```

**Comment by Sgitario** [comment]
> For a valid product_id,
> 
> ```
> 
> In [32]: product_id="OpenShift-dedicated-metrics"
> 
> In [33]: cluster=app.rhsm_subscriptions.create_mock_payg_cluster(product_id=product_id)
> 2024-04-11 14:32:13.490 [    INFO] [root] Adding payg events of org 17790802 for clowder_smoke env ...
> 2024-04-11 14:32:13.514 [    INFO] [root] POST request response url: http://swatch-tally-service:8000/api/rhsm-subscriptions/v1/internal/rpc/tally/events, status_code: 200
> 2024-04-11 14:32:13.515 [    INFO] [root] Response: {
>     "detail": "Events saved"
> }
> 2024-04-11 14:32:13.515 [    INFO] [root] OpenShift-dedicated-metrics PAYG Cluster 9afb8efe-be7a-4489-a04e-6b3a3574d2bf created successfully!!
> 
> In [34]: application.rhsm_subscriptions.sync_tally_hourly(
>     ...:         hours=3, wait=True, product_id=product_id, perform_metering=False
>     ...:     )
> 2024-04-11 14:32:24.574 [    INFO] [iqe.base.rest_client] REST: METHOD=GET, request_id=None, params=[('granularity', 'Daily'), ('beginning', '2024-04-10T14:07:17.830047+00:00'), ('ending', '2024-04-11T14:32:24.552498+00:00')]
> 2024-04-11 14:32:24.575 [    INFO] [root] {'date': datetime.datetime(2024, 4, 11, 0, 0, tzinfo=tzlocal()),
>  'has_data': False,
>  'value': 0}
> 2024-04-11 14:32:24.587 [    INFO] [root] POST request response url: http://swatch-tally-service:8000/api/rhsm-subscriptions/v1/internal/tally/hourly?org=17790802&start=2024-04-11T12%3A00%3A00Z&end=2024-04-11T15%3A00%3A00Z, status_code: 204
> 2024-04-11 14:32:24.587 [    INFO] [root] Not valid json output. Returning plain text message.
> 2024-04-11 14:32:24.587 [    INFO] [root] from 2024-04-11T12:00:00Z to 2024-04-11T15:00:00Z.
> 2024-04-11 14:32:24.600 [    INFO] [root] Waiting for hourly tally sync completion...
> 2024-04-11 14:32:39.634 [    INFO] [iqe.base.rest_client] REST: METHOD=GET, request_id=None, params=[('granularity', 'Daily'), ('beginning', '2024-04-10T14:07:17.830047+00:00'), ('ending', '2024-04-11T14:32:39.615087+00:00')]
> 2024-04-11 14:32:39.635 [    INFO] [root] {'date': datetime.datetime(2024, 4, 11, 0, 0, tzinfo=tzlocal()),
>  'has_data': True,
>  'value': 1}
> 
> In [35]: new_systable = application.rhsm_subscriptions.search_system_table(
>     ...:         product_id=product_id, display_name_contains=cluster
>     ...:     )
> 2024-04-11 14:32:46.595 [    INFO] [iqe.base.rest_client] REST: METHOD=GET, request_id=None, params=[('limit', 100), ('offset', 0), ('display_name_contains', '9afb8efe-be7a-4489-a04e-6b3a3574d2bf')]
> 2024-04-11 14:32:46.596 [    INFO] [root] [{'id': '36f4f918-78b8-43b7-b6ef-67a0a5f76f37', 'instance_id': '9afb8efe-be7a-4489-a04e-6b3a3574d2bf', 'display_name': 'automation_osd_cluster_9afb8efe-be7a-4489-a04e-6b3a3574d2bf', 'billing_provider': 'red hat', 'measurements': [1.0, 1.0], 'last_seen': datetime.datetime(2024, 4, 11, 13, 0, tzinfo=tzlocal()), 'last_applied_event_record_date': datetime.datetime(2024, 4, 11, 14, 32, 13, 507010, tzinfo=tzlocal()), 'number_of_guests': 0, '_labeled_measurements': {'Cores': 1.0, 'Instance-hours': 1.0}}]
> 
> In [36]: export = application.rhsm_subscriptions.create_swatch_export_request(
>     ...:         filters={"product_id": f"{product_id}"}, resource="instances"
>     ...:     )
> 2024-04-11 14:32:54.984 [    INFO] [iqe.base.rest_client] REST: METHOD=POST, request_id=None, params=[]
> 2024-04-11 14:32:54.985 [    INFO] [root] Waiting for export request with id e416fdd2-0274-42cd-bd2a-6aac3af28887 to process
> 2024-04-11 14:32:54.985 [    INFO] [root] {'completed_at': None,
>  'created_at': datetime.datetime(2024, 4, 11, 14, 32, 54, 977937, tzinfo=tzlocal()),
>  'expires_at': datetime.datetime(2024, 4, 18, 14, 32, 54, 977923, tzinfo=tzlocal()),
>  'format': 'json',
>  'id': 'e416fdd2-0274-42cd-bd2a-6aac3af28887',
>  'name': 'IACHMOpwwG',
>  'sources': [{'application': 'subscriptions',
>               'filters': {'product_id': 'OpenShift-dedicated-metrics'},
>               'id': 'cbe3bcdb-3cbe-4f31-be2a-f979e0a30f09',
>               'resource': 'instances',
>               'status': 'pending'}],
>  'status': 'pending'}
> ```
> 
> The export request is stuck in pending state, I see log in export service , but I see no logs in tally-service pod, nor in subscription service pods Export pod logs:
> 
> ```
> {"level":"debug","@timestamp":"2024-04-11T14:32:54.9838Z","caller":"exports/request_resources.go:92","msg":"sending kafka message to the producer","request_id":"export-service-service-57496c48d9-bkw9t/eroqxBeKT2-000017","org_id":"17790802","export_id":"e416fdd2-0274-42cd-bd2a-6aac3af28887"}
> {"level":"info","@timestamp":"2024-04-11T14:32:54.9839Z","caller":"exports/request_resources.go:94","msg":"sent kafka message to the producer: platform.export.requests[-1]@0","request_id":"export-service-service-57496c48d9-bkw9t/eroqxBeKT2-000017","org_id":"17790802","export_id":"e416fdd2-0274-42cd-bd2a-6aac3af28887"}
> {"level":"info","@timestamp":"2024-04-11T14:33:15.0129Z","caller":"logger/logger.go:95","msg":"","protocol":"HTTP/1.1","request":"GET","path":"/api/export/v1/exports/e416fdd2-0274-42cd-bd2a-6aac3af28887/status","latency":0.00324921,"status":200,"size":372,"request_id":"export-service-service-57496c48d9-bkw9t/eroqxBeKT2-000018"}
> ```

Can you spin up an EE namespace for me with the steps you did to reproduce the issue and/or change the logging level to DEBUG? If you don't see any logs in the tally service, it must be that the event was not consumed.

**Review by san7ket — APPROVED** [review]
```
In [11]: export = application.rhsm_subscriptions.create_swatch_export_request(
    ...:         filters=filters, resource="instances"
    ...:     )
2024-04-16 16:07:09.614 [    INFO] [root] Using <function client_obj_maker.<locals>.make_obj at 0x7f221fa9e4c0> object....with url http://export-service-service.ephemeral-oprp4p.svc:8000/api/export/v1 and verify_ssl set to False
2024-04-16 16:07:09.615 [    INFO] [iqe.base.auth] Setting auth_type to identity
2024-04-16 16:07:09.626 [    INFO] [iqe.base.rest_client] REST: METHOD=POST, request_id=None, params=[]
2024-04-16 16:07:09.628 [    INFO] [root] Waiting for export request with id 802328d6-e3e8-4b38-ad42-ad21e9c631f9 to process
2024-04-16 16:07:09.628 [    INFO] [root] {'completed_at': None,
 'created_at': datetime.datetime(2024, 4, 16, 16, 7, 9, 624311, tzinfo=tzlocal()),
 'expires_at': datetime.datetime(2024, 4, 23, 16, 7, 9, 624298, tzinfo=tzlocal()),
 'format': 'json',
 'id': '802328d6-e3e8-4b38-ad42-ad21e9c631f9',
 'name': 'kiZDUsEDFw',
 'sources': [{'application': 'subscriptions',
              'filters': {'beginning': '2024-04-14T00:00:00Z',
                          'product_id': 'RHEL for x86'},
              'id': '27a4d8a6-aee8-4c3f-987e-de307d97f9b7',
              'resource': 'instances',
              'status': 'pending'}],
 'status': 'pending'}
2024-04-16 16:07:29.655 [    INFO] [iqe.base.rest_client] REST: METHOD=GET, request_id=None, params=[]

In [12]: data, meta = application.rhsm_subscriptions.download_export_to_json(export.id)
2024-04-16 16:07:32.171 [    INFO] [iqe.base.rest_client] REST: METHOD=GET, request_id=None, params=[]
2024-04-16 16:07:32.176 [    INFO] [iqe.base.rest_client] REST: METHOD=GET, request_id=None, params=[]
/usr/lib64/python3.9/tarfile.py:2239: RuntimeWarning: The default behavior of tarfile extraction has been changed to disallow common exploits (including CVE-2007-4559). By default, absolute/parent paths are disallowed and some mode bits are cleared. See https://access.redhat.com/articles/7004769 for more details.
  warnings.warn(

In [13]: data
Out[13]: 
[{'id': 'c6f33948-add1-4bb3-abfc-e345edba908f',
  'instance_id': '6f019fa9-4859-4924-93b8-7ff21838d390',
  'display_name': 'physical_mockvniigbgn.test.org',
  'measurements': [{'metric_id': 'Sockets', 'value': 2.0}],
  'guests': [{'inventory_id': '40bf8bcc-42d6-4af9-99aa-b668091b7ce2',
    'insights_id': 'b89a3506-f090-4228-b223-6e3c16042dfd',
    'display_name': 'virtual_mockrcmsikfm.test.edu',
    'subscription_manager_id': 'b1da9a7d-f103-47f4-9f41-6519062bb955',
    'hardware_type': 'VIRTUALIZED',
    'last_seen': '2024-04-16T16:06:05.504977Z',
    'is_unmapped_guest': False,
    'is_hypervisor': False},
   {'inventory_id': '58142638-737f-4c21-9f72-9d1561f27861',
    'insights_id': '3359e07f-3367-47a0-b2a4-594a2c92f6a3',
    'display_name': 'virtual_mocksuebunxg.example.info',
    'subscription_manager_id': '09fad492-50e6-42b6-974d-f5c90e69f65b',
    'hardware_type': 'VIRTUALIZED',
    'last_seen': '2024-04-16T16:06:05.433303Z',
    'is_unmapped_guest': False,
    'is_hypervisor': False}],
  'last_seen': '2024-04-16T16:06:02.189588Z',
  'number_of_guests': 2,
  'category': 'hypervisor',
  'subscription_manager_id': '6cd44927-2063-4385-9cc7-d0c63b5cf3c7',
  'inventory_id': '6f019fa9-4859-4924-93b8-7ff21838d390'}]


```

File: src/main/resources/application-worker.yaml [hunk 1]
```diff
@@ -106,3 +107,21 @@ rhsm-subscriptions:
...
+    url: ${clowder.privateEndpoints.export-service-service.url:http://localhost:10000}
+    truststore: ${clowder.privateEndpoints.export-service-service.trust-store-path}
+    truststore-password: ${clowder.privateEndpoints.export-service-service.trust-store-password}
+    truststore-type: ${clowder.privateEndpoints.export-service-service.trust-store-type}
+    psk: ${SWATCH_EXPORT_PSK:placeholder}
+  export:
+    tasks:
+      topic: ${SUBSCRIPTION_EXPORT_TOPIC}
+      kafka-group-id: swatch-tally-export
+spring.jpa.show-sql: true
```
Comment (kartikshahc) [inline src/main/resources/application-worker.yaml:120]: Seems like we are intentionally showing this logs in prod logs as well?

**Review by kartikshahc — APPROVED** [review]
LGTM. Left one question in the comment and also for me it didn't create any file under export-service/tmp/minio. I will check with you @Sgitario on that but other steps worked fine.

**Pull request merged**